src/Kafka/ConsumerGroups/components/DeleteConsumerGroup.test.tsx

```diff
@@ -0,0 +1,57 @@
+import { composeStories } from "@storybook/testing-react";
+import { render, waitForI18n } from "../../../test-utils";
+import * as stories from "./DeleteConsumerGroup.stories";
+import { userEvent } from "@storybook/testing-library";
+
+const {
+  DeleteModalWhenConsumerStateIsStable,
+  DeleteModalWhenConsumerStateIsNotStable,
+} = composeStories(stories);
+
+describe("Consumer group delete modal", () => {
+  it("Delete modal when the consumer group state is equal to stable", async () => {
+    const onClose = jest.fn();
+    const onDeleteConsumer = jest.fn();
+
+    const comp = render(
+      <DeleteModalWhenConsumerStateIsStable
```

Comment:
A shorter name could be better. Try to use DeleteStableConsumerGroup/DeleteUnstableConsumerGroup or AllowConsumerGroupDeletion/DenyConsumerGroupDeletion